crates/proto/src/error.rs

```diff
@@ -345,6 +345,29 @@ pub enum ProtoErrorKind {
     NativeCerts,
 }
 
+/// Data needed to process a SOA-record-based referral.
+#[derive(Clone, Debug)]
+pub struct ForwardData {
+    /// Name
+    pub name: Name,
+    /// No records found?
+    pub is_no_records_found: bool,
```

Comment:
The `is_` prefix feels unnecessary/unidiomatic to me?

Reply:
I think it gives some consistency as `is_no_records_found` is already a method in several error types